cmake/modules/AddSwift.cmake

```diff
@@ -573,6 +573,9 @@ function(_add_swift_runtime_link_flags target relpath_to_lib_dir bootstrapping)
     else()
       message(FATAL_ERROR "Unknown BOOTSTRAPPING_MODE '${ASRLF_BOOTSTRAPPING_MODE}'")
     endif()
+  else()
+    target_link_directories(${target} PRIVATE
+      ${SWIFT_PATH_TO_SWIFT_SDK}/usr/lib/swift/Windows/x86_64)
```

Comment:
Can this be
```
target_link_directories(${target} PRIVATE
      ${SWIFT_PATH_TO_SWIFT_SDK}/usr/lib/swift/${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_LIB_SUBDIR})
```
?

Reply:
No, it cannot.  We cannot use those variables because we explicitly do not build the standard library when we build the toolchain on Windows.

Reply:
Just `SWIFT_HOST_VARIANT_ARCH` then?